doc/mkdocs/mkdocs.yml

```diff
@@ -150,6 +151,7 @@ nav:
         - 'operator value_t': api/basic_json/operator_value_t.md
         - 'operator[]': api/basic_json/operator[].md
         - 'operator=': api/basic_json/operator=.md
+        - 'operator<=>': api/basic_json/operator_spaceship.md
```

Comment:
The order of these operators is highly unintuitive. Should we order them strictly lexicographically, logically or a (better) mix of both? For example, `operator<<` is interspersed with `operator<`, etc.

Reply:
You're right. I guess the best order could be the one on https://json.nlohmann.me/api/basic_json/

Reply:
I'm trying a new order based on that.